projects/dvl-fw/src/lib/shared/visualization.ts

```diff
@@ -26,6 +26,7 @@ export interface Visualization {
   properties: { [key: string]: any };
   graphicSymbols: { [slot: string]: GraphicSymbol };
   readonly description?: string;
+  shortDescriptionText?: string;
```

Comment:
You don't need to compute short description text, you can slice in the template,

Reply:
removed it, now slicing is being done from template